tests/integration/cqlengine/management/test_compaction_settings.py

```diff
@@ -60,7 +60,7 @@ class SizeTieredCompactionChangesDetectionTest(Model):
 
             __options__ = {'compaction': {'class': 'org.apache.cassandra.db.compaction.SizeTieredCompactionStrategy',
                                           'bucket_high': '20',
-                                          'bucket_low': '10',
+                                          'bucket_low': '0.5',
                                           'max_threshold': '200',
                                           'min_threshold': '100',
                                           'min_sstable_size': '1000',
```

Comment:
Why did you need to change this? It should be explained in the commit message.

Reply:
Added info to commit, scylla does not tolerate value more than `1`.